### PR TITLE
update takeoff location for SIH

### PR DESCRIPTION
**Pull request by Claudio-Chies**
See PR https://github.com/PX4/PX4-Autopilot/pull/23363
update how the takeoff location is set for SIH

also ran prettier

### CODE REVIEW

File: en/sim_sih/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ To display the simulated vehicle:
      On macOS this would be `/dev/tty.usbmodem1`.
    - `-b` to set the serial baud rate to `2000000`.
    - `-o` to start jMAVSim in _display Only_ mode (i.e. the physical engine is turned off and jMAVSim only displays the trajectory given by the SIH in real-time).
-   - add a flag `-a` to display an aircraft or '-t' to display a tailsitter.
+   - add a flag `-a` to display an aircraft or ’-t’ to display a tailsitter.
```
Comment (hamishwillee) [inline en/sim_sih/index.md:96]: This would be done using backticks like in the `a`, as that makes it code style.

File: en/sim_sih/index.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ The vehicle can be observed moving in jMAVSim, and on the QGC _Fly_ view.
 
 SIH can be run as SITL (Software-In-The-Loop) from v1.14.
 What this means is that the simulation code is executed on the laptop/computer, similar to Gazebo or jMAVSim.
-In this case you don't need the flight controller hardware.
+In this case you don’t need the flight controller hardware.
```
Comment (hamishwillee) [inline en/sim_sih/index.md:108]: I would have reverted this. We don't use the curly apostrophes anywhere else in the docs.

File: en/sim_sih/index.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ To run the SIH, you will need a:
 - QGroundControl for flying the vehicle via GCS.
 - Development computer for visualizing the virtual vehicle (optional).
 
-From PX4 v1.14 you can run SIH "as SITL", in which case a flight controller is not required.
+From PX4 v1.14 you can run SIH “as SITL”, in which case a flight controller is not required.
```
Comment (hamishwillee) [inline en/sim_sih/index.md:50]: This was correct before.

File: en/sim_sih/index.md [hunk 4]
```diff
@@ -151,8 +151,24 @@ To display the vehicle in jMAVSim during SITL mode, enter the following command
 The dynamic models for the various vehicles are:
 
 - Quadrotor: [pdf report](https://github.com/PX4/PX4-user_guide/raw/main/assets/simulation/SIH_dynamic_model.pdf).
-- Fixed-wing: Inspired by the PhD thesis: "Dynamics modeling of agile fixed-wing unmanned aerial vehicles." Khan, Waqas, supervised by Nahon, Meyer, McGill University, PhD thesis, 2016.
-- Tailsitter: Inspired by the master's thesis: "Modeling and control of a flying wing tailsitter unmanned aerial vehicle." Chiappinelli, Romain, supervised by Nahon, Meyer, McGill University, Masters thesis, 2018.
+- Fixed-wing: Inspired by the PhD thesis: “Dynamics modeling of agile fixed-wing unmanned aerial vehicles.” Khan, Waqas, supervised by Nahon, Meyer, McGill University, PhD thesis, 2016.
+- Tailsitter: Inspired by the master’s thesis: “Modeling and control of a flying wing tailsitter unmanned aerial vehicle.” Chiappinelli, Romain, supervised by Nahon, Meyer, McGill University, Masters thesis, 2018.
+
+### Set Custom Takeoff Location
```
Comment (hamishwillee) [inline en/sim_sih/index.md:157]: This section is excellent thank you.

File: en/sim_sih/index.md (hunk 3 above)
Comment (Claudio-Chies) [reply en/sim_sih/index.md:50]: should i revert it, and change it back?

**Comment by hamishwillee** [comment]
@Claudio-Chies I can't find my original comment here, which was essentially "please don't merge without my review" - I work 2 days per week on this, starting tomorrow, so it can take up to a week for review. Sometimes I get in faster, and I'm sorry it took too long - the system is supposed to require approval.

Normally it doesn't matter too much, but because we're coming up to a release I'm holding back stuff that isn't going into v1.15.

Anyway, the question is, will this change go into PX4 v1.15? if the answer is yes, can you update https://docs.px4.io/main/en/releases/1.15.html with a release note and revert the tiny errors you injected above.

If it is not, then I'll fix this all up tomorrow and split out v1.15. (but some text for a release note in https://docs.px4.io/main/en/releases/main.html would be appreciated, which I will merge later.

Thanks very much

**Comment by Claudio-Chies** [comment]
@hamishwillee Good to know, will keep it in mind for the next PR.
i'm not yet that familiar with the release schedule, it parent PR (https://github.com/PX4/PX4-Autopilot/pull/23363) has been merged to main, which i assume means it wont go into 1.15.

**Comment by hamishwillee** [comment]
Thanks @Claudio-Chies. Stuff does into main by default and then some subset gets pushed into the release branch. However it is a  great mystery to all concerned about how the decision is made to decide some stuff goes in and some stuff does not. An even bigger mystery, when the release will happen. We're only one month after the expect date, so based on my experience, sometime in the next 6 months to a year. If I sound bitter, it's because it makes my life harder that we can't manage this better. Grrr.

I will assume that this is not being backported - it is only going into main.

**Comment by hamishwillee** [comment]
Thanks @Claudio-Chies . FYI I have split off 1.15 before this went in. https://github.com/PX4/PX4-user_guide/pull/3278 does a bit of tidy work related to this and adds a release note.

**Comment by Claudio-Chies** [comment]
Thanks @hamishwillee  for the tidying!

**Pull request merged**